src/Sulu/Bundle/SnippetBundle/Content/SingleSnippetSelection.php

```diff
@@ -88,7 +100,12 @@ private function resolveSnippet(PropertyInterface $property)
 
         /** @var PageBridge $page */
         $page = $property->getStructure();
+
         $webspaceKey = $page->getWebspaceKey();
+        if ($this->requestAnalyzer) {
+            $webspaceKey = $this->requestAnalyzer->getWebspace()->getKey();
```

Comment:
This would be a bc break and can cause now that a false default snippet is used instead of the expected structure webspace default snippet. What is the case? Is the `getWebspaceKey` not correctly returned by by articles? We should basically avoid accessing request specific things inside a content type as this can end in unexpected results.

Reply:
What we could do is adding a fallback if no webspace key is returned for the structure but not sure about it 🤔

Reply:
Another possibility would be to just call `$articleBridge->setWebspaceKey()` somewhere